include/eve/module/core/constant/ieee_constant.hpp

```diff
@@ -70,6 +70,7 @@ namespace eve
 
   namespace _
   {
+    struct real16 { constexpr real16(auto v) : value(v) {} eve::float16_t value; };
```

Comment:
Why changing this file here while there is another branch for that ?

Reply:
sans ça j'avais des pb de compil